### DISCUSSION TITLE
weather plugin

### DISCUSSION THREAD

**Discussion by johe123qwe** [n3d1117/chatgpt-telegram-bot#411] [Q&A] [answered]
I added plugin: weather in .env, I don't know how to use it to query weather. How should I use it? Thanks. 🙂

**Comment by noriellecruz** [comment]
this should be posted in the discussions tab

**Comment by n3d1117** [comment, marked as answer]
Hey @johe123qwe, make sure that your `.env` file has the `PLUGINS=weather` line and that you are using a plugins-compatible OpenAI model. Then just ask i.e. `What's the weather for tomorrow in Rome`
